lib/oli/delivery/sections/updates.ex

```diff
@@ -214,7 +214,16 @@ defmodule Oli.Delivery.Sections.Updates do
           scoring_strategy_id: r.scoring_strategy_id,
           slug: Oli.Utils.Slug.generate("section_resources", r.title),
           inserted_at: {:placeholder, :timestamp},
-          updated_at: {:placeholder, :timestamp}
+          updated_at: {:placeholder, :timestamp},
+          collab_space_config: r.collab_space_config,
+          max_attempts:
+            if is_nil(r.max_attempts) do
+              0
+            else
+              r.max_attempts
+            end,
```

Comment:
I prefer the short way of writing the `IF` statement, such as: 

`if is_nil(r.max_attempts), do: 0, else: r.max_attempts`

but that's just a personal taste!

Reply:
@simonchoxx what about: `r.max_attempts || 0`